onedark.vifm

```diff
@@ -0,0 +1,30 @@
+" gruvbox color scheme approximation for vifm
+
+" Reset all styles first
+highlight clear
+
+highlight Border	cterm=none	ctermfg=035	ctermbg=015
+
+highlight TopLine	cterm=none	ctermfg=002	ctermbg=015
+highlight TopLineSel	cterm=bold	ctermfg=002	ctermbg=015
+
+highlight Win		cterm=none	ctermfg=250	ctermbg=default
+"highlight OtherWin  cterm=none	ctermfg=223	ctermbg=236
```

Comment:
if you dont need this instead of commenting it out we could also remove the line entirely

Reply:
That line is originally commented out in the `gruvbox` theme. You might want to remove it from there as well.